sdcm/cluster_k8s/__init__.py

```diff
@@ -464,6 +468,19 @@ def wait_for_pods_readiness(self, count: Optional[int] = None):
         if result.stdout.count('condition met') != count:
             raise RuntimeError('Not all nodes reported')
 
+    def statefulsets(self, namespace=None):
+        if namespace is None:
+            namespace = self.namespace
+        return KubernetesOps.list_statefulsets(self.k8s_cluster, namespace=namespace)
+
+    def rollout_restart(self):
+        statefulsets = self.statefulsets()
+        self.k8s_cluster.kubectl("rollout restart statefulset", namespace=self.namespace)
+        for statefulset in statefulsets:
+            self.k8s_cluster.kubectl(f"rollout status statefulset/{statefulset.metadata.name} "
+                                     f"-n scylla --watch=true --timeout={self.pod_readiness_timeout}m")
```

Comment:
Please, use `namespace` parameter for `kubectl()` method instead of putting it to the cmd.

Reply:
fixed